### PR TITLE
Add ProxySQL Dashboard json

### PR DESCRIPTION
**Pull request by TasdidurRahman**
Signed-off-by: TasdidurRahman <tasdid@appscode.com>

### CODE REVIEW

File: proxysql/proxysql-summary.json [hunk 1]
```diff
@@ -0,0 +1,2410 @@
...
+        "2d",
+        "7d",
+        "30d"
+      ]
+    },
+    "timezone": "",
+    "title": "KubeDB / ProxySQL / Summary",
+    "uid": "VsnOgk2Hnk",
+    "version": 9
+  }
```
Comment (spectro30) [inline proxysql/proxysql-summary.json:2410]: add newline at the end of all files

File: proxysql/proxysql-database.json [hunk 2]
```diff
@@ -0,0 +1,7812 @@
+{
```
Comment (spectro30) [inline proxysql/proxysql-database.json:1]: add namespace if not present

File: proxysql/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+### Using Dashboards
+
+#### Create ProxySQL Metrics Configurations
+
+At first, you have to create a `MetricsConfiguration` object for ProxySQL CR. This `MetricsConfiguration` object is used by Panopticon to generate metrics for the ProxySQL instances.
+
+Run the below command to create the `MetricsConfiguration` object:
+
+```bash
```
Comment (spectro30) [inline proxysql/README.md:76]: pull this PRs changes

**Review by spectro30 — CHANGES_REQUESTED** [review]

File: proxysql/proxysql-summary.json (hunk 1 above)
Comment (TasdidurRahman) [reply proxysql/proxysql-summary.json]: added

File: proxysql/README.md (hunk 3 above)
Comment (TasdidurRahman) [reply proxysql/README.md:76]: pulled

File: proxysql/proxysql-database.json (hunk 2 above)
Comment (TasdidurRahman) [reply proxysql/proxysql-database.json:1]: added

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**